services/postingService.js

```diff
@@ -0,0 +1,14 @@
+const { postingDao } = require('../models')
+
+const createPost = async (userId, postData, image) => {
+
+    const post = await postingDao.createPost(userId)
+    const  postInfo  = await postingDao.createPostInformation(postData, post.insertId, image.location);
+
+    await postingDao.creatPostProductInformation(postData, postInfo.insertId)
+    await postingDao.creatHashtag(postData, postInfo.insertId)
```

Comment:
**현재 posting을 생성하는 과정이 다음과 같습니다.**
1. post 생성
2. postInformation 생성
3. postProductInformation
4. hashTag

지금 코드도 잘 작성해 주셨습니다! 다만, 이렇게 4 종류의 데이터를 하나의 요청에서 처리함에 있어서, 
현재 4개의 요청을 각각 보내고 있는 요청을 하나의 요청으로 묶어서 처리하는 transaction 개념을 고려해보면 좋을 것 같습니다.

공식문서 참고 : https://orkhan.gitbook.io/typeorm/docs/transactions
```javascript
// create a new query runner
const queryRunner = dataSource.createQueryRunner()

// establish real database connection using our new query runner
await queryRunner.connect()

// now we can execute any queries on a query runner, for example:
await queryRunner.query("SELECT * FROM users")

// we can also access entity manager that works with connection created by a query runner:
const users = await queryRunner.manager.find(User)

// lets now open a new transaction:
await queryRunner.startTransaction()

try {
    // execute some operations on this transaction:
    await queryRunner.manager.save(user1)
    await queryRunner.manager.save(user2)
    await queryRunner.manager.save(photos)

    // commit transaction now:
    await queryRunner.commitTransaction()
} catch (err) {
    // since we have errors let's rollback changes we made
    await queryRunner.rollbackTransaction()
} finally {
    // you need to release query runner which is manually created:
    await queryRunner.release()
}
```

Reply:
감사합니다!
말씀하신 트랜잭션 적용 후 통신 테스트 진행하였습니다.
확인 부탁드립니다.